projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -210,9 +224,11 @@ public function get_item_schema() {
 	 * Sets up a menu item for consumption by Calypso.
 	 *
 	 * @param array $menu_item Menu item.
+	 * @param bool  $admin_is_parent Check if the parent menu is admin.php.
+	 *
 	 * @return array Prepared menu item.
 	 */
-	private function prepare_menu_item( array $menu_item ) {
+	private function prepare_menu_item( array $menu_item, $admin_is_parent ) {
```

Comment:
If we can't move the above code block into `prepare_menu_item_url()` and `$admin_is_parent` stays around as a parameter, let's also give it a default value, just in case.